code/controllers/subsystem/blackbox.dm

```diff
@@ -133,7 +133,7 @@ SUBSYSTEM_DEF(blackbox)
 		if(FREQ_SUPPLY)
 			record_feedback("tally", "radio_usage", 1, "supply")
 		if(FREQ_CENTCOM)
-			record_feedback("tally", "radio_usage", 1, "centcom")
+			record_feedback("tally", "radio_usage", 1, "head office")

```

Comment:
Keep this as `centcom`. This is for feedback logging and may break if you change it.